loader/loader.go

```diff
@@ -182,11 +186,15 @@ func (l *loader) GetMatchingReleasePlans(ctx context.Context, cli client.Client,
 // If no previous Release is found, a NotFound error is returned.
 func (l *loader) GetPreviousRelease(ctx context.Context, cli client.Client, release *v1alpha1.Release) (*v1alpha1.Release, error) {
 	releases := &v1alpha1.ReleaseList{}
-	err := cli.List(ctx, releases,
-		client.InNamespace(release.Namespace),
-		client.MatchingFields{"spec.releasePlan": release.Spec.ReleasePlan})
-	if err != nil {
-		return nil, err
+	if release.Namespace != "" {
```

Comment:
Same as above.

Reply:
I'm not sure this function really needs a change. Can namespace really be nil?

Reply:
You are right. It's unnecessary. Thanks for pointing out.